### ISSUE TITLE
Extend REST API to use Seam Carving

### ISSUE THREAD

**Issue by zavolokas** [zavolokas/InpaintingSite#8]
In some cases Seam Carving can provide quite good results.
The goal is to extend REST API to process images using Seam Carving.

**Comment by ferib** [comment]
Right now we have one API (`api/inpaint`) that is used to process the inpaint algorithm.
Since the Seam Carving is similar, should we add a parameter that checks which algorithm to use?

Also, got any demo that I can check out?

**Comment by zavolokas** [comment]
Yep, it would be a good start to run one or another algorithm based on `POST api/inpaint` request payload. It can be a JSON:
```json
{
    "methods": [
          {
                "name": "SeamCarving",
                "settings": {
                 }
          }
    ]
}
```
That way it would be easy to extend it in future. For example multiple different methods or same method with different settings could be applied to one input

WRT Demo - I've created a [sample project](https://github.com/zavolokas/SeamCarving/tree/master/Samples/SimpleRemove) for the SeamCarving to demonstrate how it is used.
